specs/BrowserTaskManager.md

```diff
@@ -0,0 +1,69 @@
+# Background
+The WebView2 team has been asked to provide an API to expose the Browser Task
+Manager. At the current time, it is expected that the user will close the Task 
+Manager manually, so we do not need to provide an API to close it.
+
+In this document we describe the updated API. We'd appreciate your feedback.
+
+# Description
+We propose extending `CoreWebView2` to provide a `ShowTaskManager` method.
+This method will open a new window containing the task manager. If the task 
+manager is already opened, this method will do nothing.
+
+# Examples
+## C++: Show Task Manager
+
+``` cpp
+wil::com_ptr<ICoreWebView2_5> m_webview;
+
+// This method could be called from a menu bar item, such as 
+// [Script -> Show Task Manager]. 
+void ScriptComponent::ShowTaskManager()
```

Comment:
I also thought of including how we handle menu bar items in our sample app (see below), but not sure if it is just too much unnecessary code for a simple API.

``` cpp
#define IDM_SHOW_TASK_MANAGER 173

wil::com_ptr<ICoreWebView2_5> m_webview;

bool ScriptComponent::HandleWindowMessage(
    HWND hWnd,
    UINT message,
    WPARAM wParam,
    LPARAM lParam,
    LRESULT* result)
{
    if (message == WM_COMMAND)
    {
        switch (LOWORD(wParam))
        {
        // Example usage of how we could handle a window message that is 
        // raised from a menu (File -> Show Task Manager, for example). 
        case IDM_SHOW_TASK_MANAGER:
            CHECK_FAILURE(m_webview->ShowTaskManager());
            return true;
        }
    }

    return false;
}
```

Reply:
No don't include that extra code for window message handling in the API doc. What you have is fine. The API is not complex.